DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/merge_sort.hpp

```diff
@@ -14,9 +14,11 @@
 #include "merge.hpp"
 #include "produce.hpp"
 #include "sorting_networks.hpp"
+
+// Included from DirectProgramming/DPC++FPGA/include/
+#include "pipe_utils.hpp" 
```

Comment:
Include files should be in alphabetical order

Reply:
Not nice for batching things under '// Included from DirectProgramming/DPC++FPGA/include/'